build/SignFiles.proj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="SignFiles" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="packages\Microsoft.VisualStudioEng.MicroBuild.Core.0.4.1\build\Microsoft.VisualStudioEng.MicroBuild.Core.props" />
+
+  <PropertyGroup>
+    <BaseOutputDirectory>$(BUILD_STAGINGDIRECTORY)\vscode-arduino\extension</BaseOutputDirectory>
+    <!-- These properties are required by MicroBuild, which only signs files that are under these paths -->
+    <IntermediateOutputPath>$(BaseOutputDirectory)</IntermediateOutputPath>
+    <OutDir>$(BaseOutputDirectory)</OutDir>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FilesToSign Include="$(OutDir)\**\*.js" Exclude="$(OutDir)\**\node_modules\**\*.js">
+      <Authenticode>Microsoft400</Authenticode>
+    </FilesToSign>
+    <FilesToSign Include="$(OutDir)\out\serial-monitor-cli\win32\*.exe">
```

Comment:
What about the executables for the other serial-monitor-cli platforms?

Reply:
As far as I know, we can't apply Authenticode signatures to other kinds of executables. We could add a detached signature instead, but given that no one will be verifying it anyway, I was planning to wait for further divisional guidance. I'll add a comment.

Reply:
Did a little more digging and it looks like there are Mac certificates available. I'll see how difficult they are to use.

Reply:
There are several Mac certificates available, and at this point I don't understand the differences between them well enough to know which one to pick. I'm going to take this PR in as-is with only Windows signing, then follow up with the MicroBuild team for more guidance on Mac and Linux.